CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 - Add `lv_tabview_set_tab_name()` function - used to change a tab's name
 - Add `LV_THEME_MATERIAL_FLAG_NO_TRANSITION` and `LV_THEME_MATERIAL_FLAG_NO_FOCUS` flags
 - Reduce code size by adding: `LV_USE_FONT_COMPRESSED` and `LV_FONT_USE_SUBPX` and applying some optimization
+- Add `lv_font_load()` function - Loads a `lv_font_t` object from a binary font file
+- Add `lv_font_free()` function - Frees the memory allocated by the `lv_font_load()` function
```

Comment:
We should double check with @kisvegabor, but I'm pretty sure these changelog lines belong under 7.4, not 7.3.

Reply:
There is a conflict on the CHANGELOG.md, possibly that is the problem. I'll try to resolve.

Reply:
@embeddedt I'm not sure how the branches are used in this project. This PR should goes to the `dev` branch?

Reply:
That's correct. PRs should be sent to `dev` if they add new features, and to `master` if it's just a simple bugfix.

Reply:
> We should double check with @kisvegabor, but I'm pretty sure these changelog lines belong under 7.4, not 7.3.

Yes, I've resolved the conflict and moved these lines to 7.4.

Reply:
I've also fixed the tests, now CI passes.   
d321e0e